src/session.rs

```diff
@@ -9,11 +9,28 @@ use crate::{
     privval::SignableMsg,
     rpc::{Request, Response},
 };
+use autometrics::{
+    autometrics,
+    objectives::{Objective, ObjectiveLatency, ObjectivePercentile},
+};
 use std::{os::unix::net::UnixStream, time::Instant};
 use tendermint::{consensus, TendermintKey};
 use tendermint_config::net;
 use tendermint_proto as proto;
 
+const REQUEST_SLO: Objective =
+    Objective::new("validator-request").success_rate(ObjectivePercentile::P99_9);
+
+const SIGN_SLO: Objective = Objective::new("sign-request")
+    .success_rate(ObjectivePercentile::P99_9)
+    .latency(ObjectiveLatency::Ms500, ObjectivePercentile::P99)
+    .latency(ObjectiveLatency::Ms1000, ObjectivePercentile::P99_9);
+
+const GET_PUBKEY_SLO: Objective = Objective::new("pubkey-request")
+    .success_rate(ObjectivePercentile::P99_9)
+    .latency(ObjectiveLatency::Ms500, ObjectivePercentile::P99)
+    .latency(ObjectiveLatency::Ms1000, ObjectivePercentile::P99_9);
+
```

Comment:
ooh.. judging by https://docs.rs/autometrics/latest/autometrics/objectives/index.html, it's doing too much to my taste

i would personally not do this